data/containers/app.py

```diff
@@ -0,0 +1,24 @@
+from flask import Flask, render_template
+import os, sys, ssl
+
+app = Flask(__name__)
+message = ""
+
+@app.route("/")
+def index():
+    return render_template("index.html", message = message)
```

Comment:
IMHO the [index.html](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/data/containers/index.html) file does not use the passed message argument at all but is a static file:

    <html>
    The test shall pass
    </html>

So, the Balrog always wins :slightly_smiling_face: 

As a suggestion: either lines 12-23 are not needed or that the html template should be edited.